temoa-uiuc/data_files/04_uiuc.sql

```diff
@@ -1,48 +1,20 @@
 BEGIN TRANSACTION;
-
-
-/*
--------------------------------------------------------
-Tables in this section correspond to Sets
--------------------------------------------------------
-*/
-
--- User-defined flags to split set elements into proper subsets
-CREATE TABLE time_period_labels (
-  t_period_labels text primary key,
-  t_period_labels_desc text);
-INSERT INTO "time_period_labels" VALUES('e', 'existing vintages');
-INSERT INTO "time_period_labels" VALUES('f', 'future');
-
-CREATE TABLE technology_labels (
-  tech_labels text primary key,
-  tech_labels_desc text);
-INSERT INTO "technology_labels" VALUES('r', 'resource technology');
-INSERT INTO "technology_labels" VALUES('p', 'production technology');
-INSERT INTO "technology_labels" VALUES('pb', 'baseload production technology');
-INSERT INTO "technology_labels" VALUES('ps', 'storage production technology');
-
-CREATE TABLE commodity_labels (
-  comm_labels text primary key,
-  comm_labels_desc text);
-INSERT INTO "commodity_labels" VALUES('p', 'physical commodity');
-INSERT INTO "commodity_labels" VALUES('e', 'emissions commodity');
-INSERT INTO "commodity_labels" VALUES('d', 'demand commodity');
-
-CREATE TABLE sector_labels (
-  sector text primary key);
-INSERT INTO "sector_labels" VALUES('supply');
-INSERT INTO "sector_labels" VALUES('electric');
-INSERT INTO "sector_labels" VALUES('transport');
-INSERT INTO "sector_labels" VALUES('commercial');
-INSERT INTO "sector_labels" VALUES('university');
-INSERT INTO "sector_labels" VALUES('industrial');
-
---Tables below correspond to Temoa sets
-CREATE TABLE time_periods (
-  t_periods integer primary key,
-  flag text,
-  FOREIGN KEY(flag) REFERENCES time_period_labels(t_period_labels));
+CREATE TABLE "time_season" (
```

Comment:
Perhaps consider adding similar comments in the current iteration?

Reply:
This would be a good issue to create!